src/cli.rs

```diff
@@ -22,10 +22,78 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
+//! Command line interface
+//!
+//! This module provides a [structopt](https://docs.rs/crate/structopt) `struct` and `enum` that
+//! parse global wallet options and wallet subcommand options needed for a wallet command line
+//! interface.
+//!
+//! See the `repl.rs` example for how to use this module to create a simple command line REPL
+//! wallet application.
+//!
+//! See [`WalletOpt`] for global wallet options and [`WalletSubCommand`] for supported sub-commands.
+//!
+//! # Example
+//!
+//! ```
+//! # use bdk::bitcoin::Network;
+//! # use bdk::blockchain::esplora::EsploraBlockchainConfig;
+//! # use bdk::blockchain::{AnyBlockchain, ConfigurableBlockchain};
+//! # use bdk::blockchain::{AnyBlockchainConfig, ElectrumBlockchainConfig};
+//! # use bdk::cli;
+//! # use bdk::cli::{WalletOpt, WalletSubCommand};
```

Comment:
You can also merge those two lines with `use bdk::cli::{self, WalletOpt, WalletSubCommand}`

Reply:
fixed.

Reply:
fixed